Documentation/spelling_wordlist.txt

```diff
@@ -5,8 +5,8 @@ allocators
 alu
 ami
 analytics
-Ansible
```

Comment:
Was this really necessary? Should everyone always reorder everything based on how their editor sorts upper vs lower case?

Reply:
Is there a way this file can be sorted in a deterministic way based off of shell / editor? I suspect the words were added at the end of the file, then the file was sorted.